security/pod-vulnerability-scan.adoc

```diff
@@ -0,0 +1,27 @@
+[id="pod-vulnerability-scan"]
+= Scanning pods for vulnerabilities
+include::modules/common-attributes.adoc[]
+:context: pod-vulnerability-scan
+
+toc::[]
+
+Using the Container Security Operator (CSO), you can access vulnerability
+scan results from the {product-title} web console for container images
+used in active pods on the cluster. The CSO:
+
+* Watches containers associated with pods on all or specified namespaces
+* Queries the container registry where the containers came from for
+vulnerability information, provided an image’s registry is running image
+scanning (such as
+link:https://quay.io[Quay.io] or a
+link:https://access.redhat.com/products/red-hat-quay[Red Hat Quay] registry with Clair scanning)
+* Exposes vulnerabilities via the ImageManifestVuln object in the Kubernetes API
+
+Using the instructions here, the CSO is installed in the `marketplace-operators`
```

Comment:
This should be `openshift-operators` namespace.

Reply:
Thanks. Fixed.